assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -120,7 +125,6 @@ const CartLineItemRow = ( { lineItem } ) => {
 			</td>
 			<td className="wc-block-cart-item__quantity">
 				<QuantitySelector
-					disabled={ itemQuantityDisabled }
```

Comment:
Curious - is it worth keeping the `disabled` attribute on each of these form controls? I realise you're applying an `.is-loading` overlay/mask to disable these, but perhaps it's useful to keep the semantic `disable` as well (accessibility?). Not a blocker just clarifying.

Reply:
Agree with @haszari, if keyboard navigation allows focusing the quantity input, it should be explicitly disabled.

Reply:
True, pointer-events only prevent pointer events (🤦🏻‍♂️) thanks for pointing out, will redo them.